source/assets/stylesheets/scss/_downloads.scss

```diff
@@ -787,3 +789,33 @@ body.download {
   background:url(../images/dld-k8s.svg) no-repeat 5px 10px ;
   background-size:20px;
 }
+
+.steps-section{
+  border-top: 1px solid #ddd;
+  border-bottom: 1px solid #ddd;
+
+  h3{
+    margin: 15px auto 10px auto;
+  }
+}
+
+.recommended-list{
+  list-style: none;
+  margin: 0 0 15px 0 ;
+  padding: 0;
+}
+
+.recommended-list_child{
+  list-style: disc;
+  margin-top: -20px;
+}
+
+.link-external::after {
+  content: "";
+  background: url(/assets/images/external-link.svg) no-repeat center top;
```

Comment:
Once merged, test this to make sure it's working. The build does some things which might make this fail.